portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <style>
+	  body {
+        background-image: url('/images/background.jpg');
+        background-repeat: no-repeat;
+        background-attachment: fixed;
+        background-size: cover;
+      }
+	</style>
+  </head>
+  <body>
+    <div id="buttons">
+      <a href="contact.html">
+        <button class="button button1" style="float:right">Contact</button>
```

Comment:
The element button must not appear as a descendant of the a element.
See this for more ref: https://stackoverflow.com/questions/24837102/the-element-a-must-not-appear-as-a-descendant-of-the-button-element
Instead of using button class, you can style that element or use form

Reply:
Done. Button added at the same level of a